app/controllers/application_controller.rb

```diff
@@ -58,6 +58,10 @@ def find_rubygem
     end
   end
 
+  def latest_version
+    @latest_version ||= @rubygem.versions.most_recent
+  end
+
```

Comment:
This is because its used on `ReverseDependenciesController` and `RubygemsController` with `before_action` like `find_rubygem`, what do you recommend instead?